### ISSUE TITLE
Blend OS - KDE - doesn't boot correctly after installation is completed

### ISSUE THREAD

**Issue by Cavaire3D** [blend-os/blendOS#86]
This is an issue I have seen several other users have. The Installation GUI says the installation is completed but the UEFI boot manager sees the partitions/files as bulk instead of a fs. This prevents the OS from booting and defaults to the command line. This also happens for non-EFI systems but it uses the GRUB command line.

**Comment by CleMooling** [comment]
Yeah, I found that there's no any kernel img files in the /boot directory, and then the system is incomplete.
I tried to put some kernel img files into the directory, but still failed to start blendOS.

**Comment by touhidrahman** [comment]
I also got my machine failed booting. I was really hyped to try blendOS.

**Comment by CleMooling** [comment]
> try reinstalling from a USB flashed with etcher

I'm very sorry but the problem is not on Ventoy. I tried etcher just now and still failed to install the OS.

**Comment by Ast3risk-ops** [comment]
> > try reinstalling from a USB flashed with etcher
> 
> I'm very sorry but the problem is not on Ventoy. I tried etcher just now and still failed to install the OS.

Probably another bug, I assume you used the latest ISO when you tested etcher.

you might wanna get @RudraSwat to see this, might be something that needs fixing in the internals.
